app/dialog/dialog.js

```diff
@@ -37,6 +41,33 @@ export default class Dialog extends crsbinding.classes.ViewBase {
         });
     }
 
+    async showNestedDialog(event) {
+        const instance = this._element.querySelector("#nested-dialog").content.cloneNode(true);
+        const position = document.querySelector("#positionOption").value;
+
+        const anchor = {
```

Comment:
Break into a private property #anchor and use that here. Update the rest accordingly

Reply:
Created #anchor private property and implemented the changes were it was required.